pkg/scheduler/preemption/preemption_test.go

```diff
@@ -2663,6 +2674,7 @@ func TestFairPreemptions(t *testing.T) {
 				PreemptionStrategies: tc.strategies,
 			}, clocktesting.NewFakeClock(now))
 
+			beforeSnapshot, _ := cqCache.Snapshot(ctx)
 			snapshot, err := cqCache.Snapshot(ctx)
```

Comment:
nit: I would suggest to align namng, either call this `snapshotWorkingCopy`, or rename snapshotWorkingCopy -> snapshot. I probably prefer `snapshotWorkingCopy` more.

Reply:
Done.